### ISSUE TITLE
RAW annotation 

### ISSUE THREAD

**Issue by kghalieh** [neheller/kits21#52]
Can the annotation be used as segmentation classes?

you have 3 segmentation clases (Kidneys, tumor,cyst) but you have more in the RAW folder (Arteries,veins,unretires)  how can I make use of theses?

**Comment by neheller** [comment]
It was decided that these would not be used for the KiTS21 challenge due to quality issues, so we deleted the segmentations generated based on these for the small number of cases that had them. If you'd like to recover them, you can go back to an old commit such as [6ac7a9a1d24e0d2cdfaab56bf09aa8065b701b5e](https://github.com/neheller/kits21/tree/6ac7a9a1d24e0d2cdfaab56bf09aa8065b701b5e)

**Comment by Paddy-Xu** [comment]
Hi, thanks a lot for this information. I am actually really interested in the artery segmentation and looked into the data from the old commit. I can see that only the very small portion of the artery into the kidney is labeled. I understand this has been a long time, but I am just wondering if there was a guideline of up to which branch is labeled, e.g. renal artery, segmental artery, or is it labeled just based on if it is visible or not?

**Comment by neheller** [comment]
Yes, our policy was to label the vessels up until the celiac trunk (superior) and down until the aortic bifurcation to the common iliac arteries. The last axial frame annotated should be that in which the regions are right about to become non-overlapping.
